.streamlit/config.toml

```diff
@@ -23,7 +23,7 @@ fastReruns = false
 folderWatchBlacklist = []
 fileWatcherType = "auto"
 cookieSecret = ""
-headless = true
+headless = false
```

Comment:
this line causes Docker to fail on startup which is why it was changed in the first place 🤔